share/functions/fish_default_key_bindings.fish

```diff
@@ -2,7 +2,9 @@
 function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fish"
 	if not set -q argv[1]
 		if test "$fish_key_bindings" != "fish_default_key_bindings"
-			set -g fish_key_bindings fish_default_key_bindings # This triggers the handler, which calls us again and ensures the user_key_bindings are executed
+			# Allow the user to set the variable universally
+			set -q fish_key_bindings; or set -g fish_key_bindings
```

Comment:
I have mixed feelings about that approach. There are a lot of vars that can't be universal vars right now; either because it inherently doesn't make any sense ($PWD) or because the current code base makes them explicitly global ($dirprev). Too, expecting people to know that there is a single mention somewhere in the docs that "all vars can be universal vars unless otherwise noted" when reading documentation for, say, $fish_key_bindings is going to annoy a lot of users. So I'm still recommending that this be explicitly mentioned; especially since it's a notable change from earlier behavior.